projects/Aerogramme/default.nix

```diff
@@ -28,7 +28,8 @@
             package = lib.mkPackageOption pkgs "aerogramme" { };
           };
           # TODO: add a service definition
-          meta.broken = true;
+          # NOTE: enabling this fails with "The option `meta.broken' does not exist"
```

Comment:
Hm, I wonder how NixOS does it then, because it certainly has those meta fields...

Reply:
I don't think it was breaking before. We can investigate this later, though.